corehq/apps/reports/generic.py

```diff
@@ -943,10 +943,11 @@ def report_context(self):
             context.update(provider_function(self))
         return context
 
-    def table_cell(self, value, html=None):
+    def table_cell(self, value, html=None, zerostyle=False):
+        styled_value = '<span class="gray-zero">0</span>' if zerostyle and value == 0 else value
```

Comment:
not a fan of using html in python code, but I realize that the reports are currently structured in a way that you can't _really_ work around that. Sigh. x_x

(non-actionable...just a comment on the current structure which originally was my fault :crying_cat_face:)